xbmc/video/VideoInfoScanner.cpp

```diff
@@ -472,10 +472,16 @@ namespace VIDEO
           mediaType = MediaTypeTvShow;
         else if (info2->Content() == CONTENT_MUSICVIDEOS)
           mediaType = MediaTypeMusicVideo;
-        CServiceBroker::GetEventLog().Add(EventPtr(new CMediaLibraryEvent(
-          mediaType, pItem->GetPath(), 24145,
-          StringUtils::Format(g_localizeStrings.Get(24147).c_str(), mediaType.c_str(), URIUtils::GetFileName(pItem->GetPath()).c_str()),
-          pItem->GetArt("thumb"), CURL::GetRedacted(pItem->GetPath()), EventLevel::Warning)));
+
+        if (info2->Content() == CONTENT_TVSHOWS)
+          CServiceBroker::GetEventLog().Add(EventPtr(new CMediaLibraryEvent(
+            mediaType, pItem->GetPath(), 24145,
+            StringUtils::Format(g_localizeStrings.Get(24147).c_str(), mediaType.c_str(),CURL::GetRedacted(pItem->GetPath())), EventLevel::Warning)));
```

Comment:
Missing `,`before `CURL::GetRedacted`

Reply:
Do you mean a `,` or a whitespace? I see the `,` but I would say the whitespace is missing, no?